src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopyList = [];
+
+  const oneAction = (task) => {
+    const stateCopy =
+      stateCopyList[0] !== undefined
+        ? { ...stateCopyList[stateCopyList.length - 1] }
+        : { ...state };
+
+    switch (task.type) {
```

Comment:
Use destructuring for task
```suggestion
   const { type, ...... } = task;

    switch (type) {
```

Reply:
Thanks, solved